MANIFEST.in

```diff
@@ -1,14 +1,16 @@
-recursive-include change_this_to_name_of_package *
+recursive-include src/xem_wrapper **
```

Comment:
should the repo template be changed to have two asterisks also?

Reply:
Yes, not sure if I made this repo before I made that fix in the template, but it's updated